kinto/plugins/openid/views.py

```diff
@@ -94,6 +96,14 @@ def get_login(request):
             'description': "Provider %s requires 'email' scope" % provider,
         }
         raise_invalid(request, **error_details)
+    if prompt is not None and prompt != 'none':
+        # If the 'prompt' query string parameter is set it better be equal
+        # # to 'none' as that's the only value recognized.
+        error_details = {
+            'name': 'prompt',
+            'description': "prompt argument not recognized (%r)" % prompt,
```

Comment:
there might be a `location` key to set to `querystring` if I'm not wrong

Reply:
and I think you could even check that at the schema level actually

Reply:
something like `String(validator=colander.Regex("none"), missing=colander.drop)`

Reply:
`TypeError: __init__() got an unexpected keyword argument 'validator'`

Reply:
```
prompt = colander.SchemaNode(colander.String(),
                                 validator=colander.Regex("none"),
                                 missing=colander.drop)
```
worked